scripts/gh-release.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -e
+
+# Create a temporary branch
+git checkout -B release-tmp
+
+# Leave just the root-level files/folders needed for the release
+find -maxdepth 1 \
+	-not -name '.' \
+	-not -name '..' \
+	-not -name '.git' \
+	-not -name action.yml \
+	-not -name github-action \
+	| xargs rm -rf
+
+# Leave just the dist folder in github-action
+(cd github-action && find -maxdepth 1 \
+	-not -name '.' \
+	-not -name '..' \
+	-not -name 'dist' \
+	| xargs rm -rf )
+
+
+# Create a commit
+git add .
+git commit -m "release"
+
+# Create and push tag for full version number for the release
+git tag "$RELEASE_NAME" -f
+git push origin "$RELEASE_NAME" -f
+
+# Create a release for the current git ref using the full version number
+gh release create "$RELEASE_NAME" \
+	--title "Release $RELEASE_NAME" \
+	--generate-notes
+
+# Update and push short version number
+git tag "$TAG_NAME" -f
+git push origin "$TAG_NAME" -f
```

Comment:
Instead of moving the tag, would it make sense to trigger the build with something like `src_v1.2.3` and then set `v1.2.3` only on the build's commit? (I'm a bit wary of moving tags, but at the same time it's probably not worth investing a lot of time.)

Reply:
The current CI already moves the `major.minor` tag - this just changes the publishing/tagging mechanism 😄 

The idea is that `v0.2` always points to the latest `v0.2.x` release. If you need to pin to a specific release then you can use `v0.2.1234` etc.